src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/logs.e2e-spec.ts

```diff
@@ -38,6 +38,10 @@ describe('Logs page', () => {
     it('should show audit logs as a second tab', () => {
       logs.getTabText(1).should('eq', 'Audit Logs');
     });
+
+    it('should show daemon logs as a third tab', () => {
+      logs.getTabText(2).should('eq', 'Daemon Logs');
+    });
```

Comment:
We need to ensure that the centralized logging feature doesn't break, so what about writing a Grafana e2e test? I'm sure that @nizamial09 can point you to the proper resources.